src/config/steps/maif.coffee

```diff
@@ -1,8 +1,8 @@
 module.exports = {
-    name: 'maif',
```

Comment:
Same remark, maif step is not concerned by this PR, we have to avoid editing it.